ci/release/apply_wheel_modifications.sh

```diff
@@ -6,8 +6,12 @@
 VERSION=${1}
 CUDA_SUFFIX=${2}
 
+# pyproject.toml versions
 sed -i "s/^version = .*/version = \"${VERSION}\"/g" pyproject.toml
+
+# pyproject.toml cuda suffixes
 sed -i "s/^name = \"ucx-py\"/name = \"ucx-py${CUDA_SUFFIX}\"/g" pyproject.toml
+sed -i "s/cudf/cudf${CUDA_SUFFIX}/g" pyproject.toml
```

Comment:
```suggestion
```

Reply:
`cudf` isn't a dependency in UCX-Py, that's why this wasn't here, it's only a dependency for testing. See https://github.com/rapidsai/ucx-py/blob/branch-0.32/pyproject.toml and https://github.com/rapidsai/ucx-py/blob/branch-0.32/dependencies.yaml.

Reply:
This modifies the pyproject.toml specifically for the testing dependency on cudf. cudf is now listed in the optional test dependencies, but wasn’t previously. This change to dependencies.yaml/pyproject.toml/apply_wheel_modifications.sh fixed #946.

Reply:
Ah yes, it came later on in the list of changes and I didn't make the connection. Thanks for noticing my mistake.